torch/testing/_internal/common_jit.py

```diff
@@ -143,18 +143,78 @@ def getExportImportCopy(self, m, also_test_file=True, map_location=None):
             torch.jit.save(imported, fname)
             return torch.jit.load(fname, map_location=map_location)
 
+    def autoDiffErrorMessage(self, should_autodiff_node, nodes_not_in_diff_graph, 
+                             fusion_nodes_not_found, non_fusible_nodes_being_fused, 
+                             fusion_nodes_found, nodes_in_diff_graph):
+        err_msg = "\nFailure in testing nodes' autodifferentiation, "
+        if should_autodiff_node:
+            err_msg += "one or more nodes were expected to be autodiffed, " \
+                "but were not found in specified fusible/nonfusible " \
+                "DifferentiableGraph groups. \nSpecifically:"
+            for node in nodes_not_in_diff_graph:
+                err_msg += f"\n  {node} was not in one of the DifferentiableGraphs " \
```

Comment:
it might be nice to join up this error message for all nodes that are not in when expected to be. 
`{aten::add, aten::sum} were not in one of the DifferentiableGraphs` and the same for bucketing all the `non_fusible_nodes_being_fused` `fusion_nodes_not_found` groups